src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -127,7 +127,16 @@ protected virtual void Initialize()
             }
 
             // Bindings may use name resolution, so provide this before reading the bindings. 
-            ScriptConfig.HostConfig.NameResolver = new NameResolver();
+            var nameResolver = new NameResolver();
+
+            var dashboardString = nameResolver.Resolve("AzureWebJobsDashboard");
```

Comment:
Looks like this will also need an update to the SDK to disable the regular account. 
I think we should move away from a single static AmbientConnectionStringProvider and instead use the NameResolver on the JobHostConfiguration. Separate discussion.